service/runner.py

```diff
@@ -67,4 +81,13 @@ def run_only_once():
     for account_id in accs:
         workflow.HarvesterWorkflow.process_account(account_id)
 
-    app.logger.info(Report.write_report())
+    report = Report.write_report()
+    app.logger.info(report)
+
+    # If the harvester finishes normally, we can email the report.
+    if mail_prereqs:
```

Comment:
This appears to just be a translation of `if to is not None`. If line 58 is inserted where line 56 is, you could use `if to is not None` in all the following blocks of code rather than introducing a new variable. `mail_prereqs` would go completely in this case. Not a blocking issue.